plugins/en/qml/Keyboard_en_tablet_email.qml

```diff
@@ -110,7 +110,7 @@ KeyPad {
             LanguageKey    { id: languageMenuButton;                       anchors.left: symShiftKey.right; }
             UrlKey         { id: emailKey;    label: "@"; shifted: "@";  anchors.left: languageMenuButton.right; }
             SpaceKey       { id: spaceKey;                               anchors.left: emailKey.right; anchors.right: urlKey.left; noMagnifier: true }
-            UrlKey         { id: urlKey; label: ".com"; extended: [".com", ".co.uk", ".net", ".org", ".edu", ".gov", ".ac.uk"]; anchors.right: apostropheKey.left; }
+            UrlKey         { imgNormal: UI.imageGreyKey[formFactor]; imgPressed: UI.imageGreyKeyPressed[formFactor]; id: urlKey; label: ".com"; extended: [".net", ".org", ".edu", ".gov", ".co.uk", ".ac.uk"]; anchors.right: apostropheKey.left; }
```

Comment:
are you sure it's useful to change imgNormal and imgPressed here ? I thought I did that in UrlKey.qml directly